ambuild2/task.py

```diff
@@ -66,12 +64,6 @@ def __init__(self, pump, channel, vars):
     })
 
   def receiveTask(self, channel, message):
-    task_id = message['task_id']
-    task_type = message['task_type']
-    task_folder = message['task_folder']
-    if not task_folder:
-      task_folder = '.'
```

Comment:
this variable is not used anywhere. Even if it would be, there's the same check a few lines below (`if message['task_folder']:`), so there's no need to make the same check twice, is it?